pages/price-feeds/troubleshoot/evm.mdx

```diff
@@ -0,0 +1,35 @@
+# Troubleshoot Price Feeds
+
+This reference page is designed to help you troubleshoot common issues you may encounter when using Pyth Price Feeds on EVM chains.
+Follow the steps provided below to diagnose and resolve the issue.
+
+#### EVM revert getPrice() with `StalePrice()` or `0x19abf40e` error
+
+This error occurs when the price data of the requested asset is not updated in the last valid time period.
+The valid time period for the feed can queried using the [`getValidTime()`](https://github.com/pyth-network/pyth-crosschain/blob/main/target_chains/ethereum/sdk/solidity/AbstractPyth.sol#L22) method.
+
+To resolve this issue:
+
+- Update the price data by calling [`updatePriceFeeds()`]("https://github.com/pyth-network/pyth-crosschain/blob/main/target_chains/ethereum/sdk/solidity/AbstractPyth.sol#L88")
+  by passing the latest updateData from [Hermes](https://hermes.pyth.network/docs/#/rest/latest_vaas).
+- Another method to fetch the price is [`getPriceUnsafe()`](https://github.com/pyth-network/pyth-crosschain/blob/main/target_chains/ethereum/sdk/solidity/AbstractPyth.sol#L41)
+  If the price feed is available, the method will returnt the latest price data with timestamp of last update.
+  NOTE: `getPriceUnsafe()` method does not check the freshness of the price data.
+
+#### EVM revert getPrice() with `PriceFeedNotFound()` or `0x14aebe68` error
+
+This error occurs when the requested price is not published on the respective chain or the prices have never been updated.
```

Comment:
the first part of this sentence makes it sound like some feeds aren't available on some chains. I think the right sentence is "when the requested price feed has not been updated on-chain, or the price feed id is incorrect."